packages/@aws-cdk/cdk/lib/cloudformation/dynamic-reference.ts

```diff
@@ -25,7 +25,7 @@ export interface DynamicReferenceProps {
  * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/dynamic-references.html
  */
 export class DynamicReference extends Construct {
-  private readonly _value: string;
+  private _value: string;
```

Comment:
Should this remain readonly? I don't see it mutated anywhere.

Reply:
Yeah, it's just that it's a little silly to have it readonly. But sure.